src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -189,6 +190,8 @@ public void robotPeriodic() {
         jackButtonPressed = manipulator.getButton(Button.X);
 
         aimingLightsButtonPressed = driverLeft.getRawButton(2);
+
+        SmartDashboard.putNumber("Jerk Limit", driveBase.getJerkLimit());
```

Comment:
This does not allow changing the jerk limit from the Smart Dashboard.
Perhaps you want.
```suggestion
        driveBase.setJerkLimit(SmartDashboard.getNumber("Jerk Limit", driveBase.getJerkLimit()));
```